zmq4_reqrep_test.go

```diff
@@ -151,3 +152,178 @@ func TestReqRep(t *testing.T) {
 		})
 	}
 }
+
+func TestMultiReqRepIssue70(t *testing.T) {
+	var (
+		reqName  = zmq4.NewMsgString("NAME")
```

Comment:
```suggestion
		reqName1  = zmq4.NewMsgString("NAME1")
```
(and ditto for others)